app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -170,6 +237,11 @@ public void onActivityCreated(Bundle savedInstanceState) {
     public void onStart() {
         super.onStart();
         EventBus.getDefault().register(this);
+
+  //      autoDownload = SubscriptionFilterDialog.AutoDownload.valueOf(prefs.getString(PREF_AUTO_DOWNLOAD,STRING_NO_FILTER));
+//        keepUpdated = SubscriptionFilterDialog.KeepUpdated.valueOf(prefs.getString(PREF_KEEP_UPDATED,STRING_NO_FILTER));
+    //    autoDelete =SubscriptionFilterDialog.AutoDelete.valueOf(prefs.getString(PREF_AUTO_DELETE,STRING_NO_FILTER));
```

Comment:
Please remove code that was commented for test purposes. We do not want to have this in the code base forever.